### DISCUSSION TITLE
App Question

### DISCUSSION THREAD

**Discussion by idominiki** [teslamate-org/teslamate#1993] [Q&A]
Hello @adriankumpf,

I have a short developer question. How can I contact you?

Thanks and Greets
Dominik

**Comment by adriankumpf** [comment]
[Discussions](https://github.com/adriankumpf/teslamate/discussions) are the right place to ask questions :)

**Reply by idominiki** [reply]
Hi @adriankumpf !

I want to bring an iOS App to the AppStore, which will display Teslamate Data in a nice way.

Can I call the App TesMate? Because of the similarity name to your project, I wanted to ask you.

Thansk and Greets!
Dominik

**Reply by adriankumpf** [reply]
Hi Dominik. Sure, that's fine with me 👍
